### PR TITLE
feat: add configuration for debounce

### PR DESCRIPTION
**Pull request by a-catgirl-dev**

### CODE REVIEW

**Comment by a-catgirl-dev** [comment]
@Vaneer420 merge pls xx

**Comment by a-catgirl-dev** [comment]
@Vaneer420 please i need the commits

File: .envexample [hunk 1]
```diff
@@ -1,4 +1,6 @@
 token=BOTTOKENHERE
 vcid=IDofvoicechannelthebotshouldplaymusicin
 guildid=IDofguildthevoicechannelisin
-activity=theActivityForExample'MusicInRR!'
\ No newline at end of file
+activity=theActivityForExample'MusicInRR!'
+debounce_activation_threshold=timesUserMustSendAMessageToTriggerDebounce
```
Comment (a-catgirl-dev) [inline .envexample:5]: superseded by #11

**Comment by Vaneer420** [comment]
debounce.js exports will be imported into api.js on later date

**Review by Vaneer420 — APPROVED** [review]

**Comment by a-catgirl-dev** [comment]
> debounce.js exports will be imported into api.js on later date

bruh
ok

**Pull request merged**